docs/userGuide/syntax/attributes.mbdf

```diff
@@ -0,0 +1,65 @@
+## Classes, Attributes & Identifiers
+
+Most markdown syntax above this section supports adding classes, attributes and identifiers
+using [pandoc](https://pandoc.org/MANUAL.html) syntax without the need for a wrapper html element.
+
+The syntax is `{.class-name attribute="value" attribute=value #id}`, which is placed at different locations depending
+on the type of markdown.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+Apply classes, attributes, identifiers to block level markdown (eg. paragraphs, headings)
+by leaving a space before '{' {.text-success #attribute-example}
+
+#### heading {.text-info}
+
+--- {.border-danger}
+
+Apply the same to inline markdown (eg. bold text) by
+omitting the **space**{.text-primary .bg-light attribute=value}
```

Comment:
![image](https://user-images.githubusercontent.com/1673303/88937525-dbb25600-d2b6-11ea-9ac2-534fde70d5df.png)

mistake?

Reply:
html attributes don't require the `"quotes"` if there aren't any spaces in the value

Reply:
I didn't mean the quotes. That whole thing shouldn't be there, right?

Reply:
it's to show an example of attributes (vs classes and ids), but since these aren't visual, you'd have to use inspect element to see it.

Reply:
Can use a real or real-looking attribute name and a value instead? Otherwise it looks like a generic placeholder.

Reply:
added a comment to use inspect element